src/services/link/index.js

```diff
@@ -16,25 +16,49 @@ module.exports = function setup() {
 		lean: true,
 	};
 
+	const registrationLinkExpirationDays = app.get('registrationLinkExpirationDays') || 30;
+
 	let linkService = service(options);
 
-	function redirectToTarget(req, res, next) {
-		if (req.method === 'GET' && !req.query.target) { // capture these requests and issue a redirect
+	const hasExpired = (date) => {
+		// calculate the date the link will expire on
+		const expirationDate = new Date(date);
```

Comment:
What is (date) param, that it is required to create a date from it? Can't you just do the calculations on timestamps here?